dask_cuml/tests/test_linear_regression.py

```diff
@@ -48,7 +48,7 @@ def load_data(nrows, ncols, cached = 'data/mortgage.npy.gz'):
 
 @pytest.mark.skip(reason="Test should be run only with libcuML.so")
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
-def test_ols():
+def test_ols(datatype):
```

Comment:
This might yell at me still since it doesn't actually use the variable anywhere. Maybe comment out the parametrize for now?

Reply:
sure, one second